docs/book/start/logging-packets.md

```diff
@@ -8,88 +8,18 @@ Let's expand this program to log the traffic that is being permitted in the user
 
```

Comment:
Above it says:

```
In the previous chapter, our XDP application ran until Ctrl-C was hit and permitted all the traffic.
Each time a packet was received, the BPF program created a log entry.
Let's expand this program to log the traffic that is being permitted in the user-space application instead of the BPF program.
```

So we're already using aya-log in the previous chapter, you don't need to reintroduce it again. And you need to explain the difference between the previous chapter and the current chapter:

```
In the previous chapter our XDP program logged the string "received a packet" for each packet received. In this chapter we're going to show how to parse packets.
```

And the title of the chapter should probably be called "Parsing packets". The whole "Receiving the logs" section should probably go since we've already configured logging - we're just changing what's logged in this chapter.